src/lib/workspace/ComputationRoute.tsx

```diff
@@ -11,6 +12,14 @@ export interface Props {
 export function ComputationRoute(props: Props) {
   const { sessionState } = props;
   const { url } = useRouteMatch();
+  const dataClient = useDataClient();
+  const computations = usePromise(
+    useCallback(() => dataClient.getApps(), [dataClient])
+  );
+  const computationAppOverview = computations.value?.apps.find(
+    (app) => app.name === 'pass'
+  );
+  if (computationAppOverview == null) return null;
```

Comment:
I was also wondering if we need a `<Suspense>` for this bit?
Or are we assuming the getApps() call will always be fast?

Reply:
It should be pretty fast. I don't think this endpoint does any database lookups, but we should still handle possible slow responses. I'll add this.